File: meeting_3/fooBarV1.js
function fooBarV1(num) {
    console.log(`Foo Bar Game of ${num}`);
    for (let i = 1; i <= num; i++) {
        let output = '';
        if (i % 3 == 0) output += 'Foo';
        if (i % 5 == 0) output += 'Bar';
        if (i % 3 != 0 && i % 5 != 0) output += i.toString();
        console.log(output);
    }
}

fooBarV1(15);
fooBarV1(42);
fooBarV1(100);

File: meeting_3/fooBarV1.cpp
#include <iostream>

using namespace std;

void fooBar(int num);

int main() {    
    fooBar(15);
    fooBar(42);
    fooBar(100);
}

void fooBar(int num) {
    cout << "Foo Bar Game of " << num << endl;
    for (int i = 1; i <= num; i++) {
        string output = "";
        if (i % 3 == 0) output += "Foo";
        if (i % 5 == 0) output += "Bar";
        if (i % 3 != 0 && i % 5 != 0) output += to_string(i);
        cout << output << endl;
    }
}